master/package.py

```diff
@@ -49,6 +49,13 @@
         flunkOnFailure=False,
     ),
 
+    # Clear out old .cov and .mem files.  Note that this should be removed once we always start from scratch. 
+    steps.ShellCommand(
+        name="Clear out usr",
+        command=["/bin/bash", "-c", "find . -name *.jl.*.cov -o -name *.jl.mem | xargs rm -f"],
```

Comment:
How about just `find . -delete ...`?

Reply:
Nice.  Although you must be very careful; `find . -delete ...` will delete every file, because `-delete` appears before the criterion.  You should instead use `find . .... -delete`.